scripts/abi_bindings.sh

```diff
@@ -51,6 +52,14 @@ echo "Building Contracts"
 cd $TELEPORTER_PATH/contracts
 forge build --force --extra-output-files abi bin
 
+function convertToLower() {
+    if [ "$ARCH" = 'arm64' ]; then
+        echo $1 | perl -ne 'print lc'
```

Comment:
Honestly I have no idea what's going on with this function or why it's needed. I just moved it from `utils.sh` because this `abi_bindings.sh` is the only caller now.

Reply:
I'm fine with leaving it as is. I was just surprised to see it.